From: Felix Krull <f_krull@gmx.de>
Date: Fri, 27 Apr 2018 08:24:08 -0700
Subject: bdist-wininst-notfound

# DP: suggest installation of the pythonX.Y-dev package, if bdist_wininst
# DP: cannot find the wininst-* files.
---
 Lib/distutils/command/bdist_wininst.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/Lib/distutils/command/bdist_wininst.py b/Lib/distutils/command/bdist_wininst.py
index bbe2294..7e5adca 100644
--- a/Lib/distutils/command/bdist_wininst.py
+++ b/Lib/distutils/command/bdist_wininst.py
@@ -324,5 +324,8 @@ class bdist_wininst (Command):
         # we must use a wininst-x.y.exe built with the same C compiler
         # used for python.  XXX What about mingw, borland, and so on?
         filename = os.path.join(directory, "wininst-%s.exe" % bv)
-        return open(filename, "rb").read()
+        try:
+            return open(filename, "rb").read()
+        except IOError, msg:
+            raise DistutilsFileError, str(msg) + ', please install the python%s-dev package' % sys.version[:3]
 # class bdist_wininst
